app/code/community/Aoe/JsCssTstamp/Block/Head.php

```diff
@@ -264,39 +264,56 @@ protected function _separateOtherHtmlHeadElements(&$lines, $itemIf, $itemType, $
      */
     protected function &_prepareStaticAndSkinElements($format, array $staticItems, array $skinItems, $mergeCallback = null)
     {
+        $versionKey = Mage::getModel('aoejscsststamp/package')->getVersionKey();
```

Comment:
This function is protected. To make it work, it should be public.
Please implement this in your pull request.